src/main/java/chess/view/OutputView.java

```diff
@@ -0,0 +1,54 @@
+package chess.view;
+
+import chess.domain.piece.Piece;
+import chess.domain.piece.type.Knight;
+import chess.domain.position.Position;
+import chess.domain.position.Rank;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class OutputView {
+
+    public void printChessBoard(final Map<Position, Piece> pieces) {
+        final List<List<String>> board = sortByBoardOrder(pieces);
+        board.forEach(this::printChessRow);
+    }
+
+    private void printChessRow(final List<String> row) {
+        row.forEach(System.out::print);
+        System.out.println();
+    }
+
+    private List<List<String>> sortByBoardOrder(final Map<Position, Piece> pieces) {
+        final List<List<String>> board = new ArrayList<>();
+        for (int i = 0; i < Rank.values().length; i++) {
+            board.add(new ArrayList<>(List.of(".", ".", ".", ".", ".", ".", ".", ".")));
+        }
+
+        for (Entry<Position, Piece> entry : pieces.entrySet()) {
+            final int fileIndex = entry.getKey().getFile().getIndex() - 1;
+            final int rankIndex = 7 - (entry.getKey().getRank().getIndex() - 1);
+            final List<String> marks = board.get(rankIndex);
+            marks.set(fileIndex, convertToMark(entry.getValue()));
+        }
```

Comment:
`Map.entrySet()`을 순회하는 것 보단 `Map.keySet()`을 순회하는 것이 가독성이 더 좋다고 생각해요. (개인적인 생각입니다.)

또한, 디미터 법칙을 지키도록 변경해 볼 수 있을 것 같아요. [코드 한 줄에 점(.)을 하나만 허용했는가?](https://github.com/woowacourse/woowacourse-docs/blob/main/cleancode/pr_checklist.md#%EC%BD%94%EB%93%9C-%ED%95%9C-%EC%A4%84%EC%97%90-%EC%A0%90%EC%9D%84-%ED%95%98%EB%82%98%EB%A7%8C-%ED%97%88%EC%9A%A9%ED%96%88%EB%8A%94%EA%B0%80)
```suggestion
        for (Position position : pieces.keySet()) {
            final int fileIndex = position.getFileIndex() - 1;
            final int rankIndex = 7 - (position.getRankIndex() - 1);
            // 이하 생략...
        }
```